code/modules/telesci/gps.dm

```diff
@@ -207,7 +214,7 @@ var/const/DEATHSOUND_CHANNEL = 300
 
 /obj/item/device/gps/secure/proc/deathsound(var/turf/pos,var/dead=FALSE,num)
 	var/sound_channel = DEATHSOUND_CHANNEL + num
-	if(dead)
+	if(dead && transmitting)
```

Comment:
Seems to me that `transmitting` can never be false here. `deathsound()` is only called by `announce()`, which returns early if `trasmitting` is false.

Reply:
the reason I felt compulsion to change it was because during a round with an away mission, we sent in officers through the gateway with SPS's and I watched base. I could hear the SPS sound play, but nothing was being broadcasted. I didn't really test it though, and now that I think about it I don't recall any of the actual SPS carriers actually dying though because I also had a cloning back up of each of them and it would give me their health and status in real time (but without coordinates). I guess I'll just not worry about it for now and maybe test it and make a issue report if I find anything reproducible.